test/test_nestedtensor.py

```diff
@@ -8654,11 +8584,13 @@ def test_backward(self, device, dtype, op):
 
                     self.assertEqualNoncontigAware(grads, grads_ref)
 
-    @torch._dynamo.config.patch(capture_dynamic_output_shape_ops=True)
     @ops(
         [op for op in njt_op_db if op.supports_njt],
         allowed_dtypes=(torch.float32,),
     )
+    @torch._dynamo.config.patch(capture_dynamic_output_shape_ops=True)
+    # needed to avoid "data dependent operator: aten._local_scalar_dense.default"
+    @torch._dynamo.config.patch(capture_scalar_outputs=True)
```

Comment:
Yep saw these changes as part of the narrow PR as well haha

Though one reason for not doing it by default for all ops is if we want to get signal on which ops need which flags to work with compile, but maybe not worth the hassle.

Reply:
I think what I want to do is have a section of the upcoming NJT docs dedicated to torch.compile-based stuff, and mention the trick to add this flag as needed for data-dependent operators.